doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -0,0 +1,231 @@
+
+There is no consistent mapping of the :file:`sdk_config.h` entries to Kconfig options.
+Some libraries that were ported from nRF5 have similar Kconfig options as the :file:`sdk_config.h` entries they had in nRF5, but it is not a consistent rule.
+
+In nRF5, the same application/sample had a project file for each supported board and IDE/compiler.
+In |BMshort|, there is a single project file (consisting of :file:`CMakeLists.txt` and :file:`prj.conf`) that can be built for different boards with a different command-line instruction or by selecting a different board target in the VS Code extension.
+If necessary, Kconfig options can be specified in a different :file:`.conf` file that is then automatically appended to the default :file:`prj.conf` file, thus realizing a dedicated configuration for a specific board.
+Kconfig options appended in this way are referred to as **Kconfig fragments**.
+
+Once the application is built, flashing of the SoftDevice must be performed manually.
+This step must be performed only once and then the application can be flashed multiple times without having to reflash the SoftDevice.
+
+Memory partitioning
+===================
+
+In nRF5, memory partitioning was done using linker scripts.
+In |BMshort|, there are a few ready-made partitioning schemes that can be selected by compiling for specific board targets that cover the most common use cases.
+Partitioning can be tweaked by making simple changes to textual files called **Devicetree**, which define the layout of the memory.
+These can be edited in the board files, or applied to existing boards as **overlays**.
+
+Libraries and drivers
+*********************
+
+Whereas nRF5 supported different short range protocols such as Gazell, ESB, Ant, and NFC, those are not supported by |BMshort|.
+In general, |BMshort| support focuses on Bluetooth Low Energy.
+
+Bluetooth LE libraries
+======================
+
+Bluetooth LE features that are natively offered by the SoftDevice are mostly unchanged from the nRF5, and the SoftDevice documentation highlights any differences.
+As for the collection of Bluetooth LE libraries that were available in the nRF5, |BMshort| offers a limited subset, where each service may have slightly different API and functionality compared to their respective nRF5 implementation.
+
+The Bluetooth LE services offered in |BMshort| are the following:
+
+* Peripheral services:
+
+  * Heart Rate Monitor (peripheral)
+  * Nordic UART (NUS) (peripheral)
+  * Nordic LED Button (LBS) (peripheral)
+  * Continuous Glucose Monitor (peripheral)
+  * Battery (peripheral)
+
+* MCUMgr service (DFU service)
+* Bond Management
+* Device Information
+
+Utility libraries for Bluetooth LE are available in |BMshort|, but again, their collection may not be as complete, and their functionality and API may be slightly different than their respective nRF5 implementation.
+
+See table below for a summary of supported libraries.
```

Comment:
This list does not cover all libraries from nRF5, where some that we state no will come in the future. Should this be rewritten in a way that makes that clear? Or can we remove those that are not supported at the moment?